lightstep/resource_metric_condition.go

```diff
@@ -212,8 +198,15 @@ func getQuerySchema(includeSpansQuery bool) map[string]*schema.Schema {
 			Type:         schema.TypeString,
 			Optional:     true,
 			Computed:     true,
-			ValidateFunc: validation.StringInSlice([]string{"rate", "delta", "avg", "last"}, false),
+			ValidateFunc: validation.StringInSlice([]string{"rate", "delta", "last", "min", "max", "avg"}, false),
+		},
+		"timeseries_operator_input_window_ms": {
+			Type:     schema.TypeInt,
+			Optional: true,
+			// Duration micros must be at least 30s and an even number of seconds
```

Comment:
"an event number of seconds" or "a multiple of 30s"

Reply:
I think you could technically specify 44s and qengine would not complain. 🤷

Reply:
Oh, so it literally does mean an even number of seconds? Why not an odd number??? This question is not relevant to this PR, I'm now just curious :)

Reply:
You could have an odd number of seconds. What I'm trying to get at here is that you can't specify fractions of seconds even though the unit is milliseconds. I can see now this needs rephrasing :)
45000 is valid
44000 is valid
1000 is invalid
44500 is invalid

Reply:
this is because windows in the qengine api are sepcified in seconds (type int)

Reply:
maybe "round" in place of "even" would squash the confusion? i.e. % 1000 == 0. or micros must represent a round integer in seconds